src/Module/Xrd.php

```diff
@@ -152,6 +152,7 @@ private function printSystemJSON(array $owner)
 			]
 		];
 		header('Access-Control-Allow-Origin: *');
+		header('Vary: Accept', false);
```

Comment:
Please move this statement to line 82 which removes the need for the one on line 233 and the one on line 331.